app/presenters/publishing_api/redirect_presenter.rb

```diff
@@ -0,0 +1,39 @@
+module PublishingApi
+  class RedirectPresenter
+    attr_accessor :item, :update_type
+
+    def initialize(item, update_type: nil)
+      self.item = item
+      self.update_type = update_type || "major"
+    end
+
+    delegate :content_id, to: :item
+
+    def content
+      {
+        title: nil,
```

Comment:
Do we need to specify a nil title, as it is forbidden in [the content schema](https://github.com/alphagov/publishing-api/blob/b0cf0ebd9c25b1bb918fb9491c93a3118fb2599e/content_schemas/formats/redirect.jsonnet#L7)?